File: src/main/java/com/iog/Commands/Other/BotInfo.java
package com.iog.Commands.Other;

import com.iog.Commands.BaseCommand;
import com.iog.Main;
import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
import net.dv8tion.jda.api.interactions.commands.build.Commands;
import org.jetbrains.annotations.NotNull;

public class BotInfo extends BaseCommand {

	public BotInfo() {
		super(
			Commands.slash("botinfo", "Tells information about the bot")
		);
	}
	
	@Override
	public void run(@NotNull SlashCommandInteractionEvent event) {
		String selfId = event.getJDA().getSelfUser().getId();

		String text = "```" +
				"Rift: " + "\n" +
				"Version: " + Main.Version + "\n" +
				"ID: " + selfId + "\n" +
				"Github: " + "**https://github.com/ItsOnlyGame/Rift**" + "\n\n" +
				"You can report you issues to github\n```";

		event.reply(text).queue();
	}
}

File: src/Commands/Utils/BotInfo.ts
import { Message } from 'discord.js'
import Command from '../../Models/Command';

export default class BotInfo extends Command {
    constructor() {
        super(
            "Bot Info",
            "Bot information",
            ["bot"],
            null
        )
    }

    public async execute(message: Message, args: string[]): Promise<void> {
        const str =
            "```" +
            "Rift: " + "\n" +
            "Version: " + process.env.npm_package_version + "\n" +
            "ID: " + message.channel.client.user.id + "\n" +
            "Github: " + "https://github.com/ItsOnlyGame/Rift" + "\n\n" +
            "You can report you issues to github\n```";

        message.channel.send(str)
    }

}